databuilder/loader/generic_loader.py

```diff
@@ -0,0 +1,54 @@
+import logging
+
+from pyhocon import ConfigTree  # noqa: F401
+from typing import Optional, Any  # noqa: F401
+
+from databuilder.loader.base_loader import Loader
+
+LOGGER = logging.getLogger(__name__)
+
+CALLBACK_FUNCTION = 'callback_function'
+
+
+def log_call_back(record):
+    """
+    A Sample callback function. Implement any function follows this function's signature to fit your needs.
+    :param record:
+    :return:
+    """
+    LOGGER.info('record: {}'.format(record))
+
+
+class GenericLoader(Loader):
```

Comment:
should add a doc pr later to put example on how to use different loader / extractor . E.g I assume this one is to use with our event client?

If we use with event client, I wonder what publisher we should use in this case(e.g do we have noop publisher)?